### PR TITLE
New package: MarkGriffiths.NetTime version 3.14

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

### CODE REVIEW

File: manifests/m/MarkGriffiths/NetTime/3.14/MarkGriffiths.NetTime.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: MarkGriffiths.NetTime
+PackageVersion: "3.14"
+PackageLocale: en-US
+Publisher: Mark Griffiths
+PackageName: NetTime
```
Comment (stephengillie) [inline manifests/m/MarkGriffiths/NetTime/3.14/MarkGriffiths.NetTime.locale.en-US.yaml:8]:
```suggestion
PackageName: NetTime
PackageUrl: https://timesynctool.com/
```

**Comment by 12932** [comment]
@BoxedLight what happened to this? I'm new to Winget and was hoping NetTime would be in the package list :P

**Pull request closed without merging**